src/main/java/frc/robot/Robot.java

```diff
@@ -28,8 +34,32 @@ public void robotInit() {
     // Instantiate our RobotContainer.  This will perform all our button bindings, and put our
     // autonomous chooser on the dashboard.
     m_robotContainer = new RobotContainer();
+		// register subsystems into a list to minimize redundancy if logging
+		// inputs/outputs for all
+		inst = NetworkTableInstance.create();
+
+		// start a NT4 client
+		inst.startClient4("robot network table client");
+
+		// connect to a roboRIO with team number TEAM
+		inst.setServerTeam(8248);
+
+		// connect to a specific host/port
+		inst.setServer("10.0.0.2", NetworkTableInstance.kDefaultPort4);
+		USB2 = inst.getTable("photonvision").getSubTable("USB2.0_PC_CAMERA");
+		if (OPEN_TAG_WINDOW) {
```

Comment:
Is this supposed to be a dynamically-controlled conditional? As it stands currently, this code would get compiled/optimized out (and is therefore dead code).